mempool/s3_memory_pool.c

```diff
@@ -342,7 +342,7 @@ void *mempool_getbuffer(MemoryPoolHandle handle, size_t expected_buffer_size) {
     snprintf(log_mem_stats, sizeof(log_mem_stats), log_memool_stats,
              pool->mempool_item_size, pool->free_bufs_in_pool,
              pool->number_of_bufs_shared, pool->total_bufs_allocated_by_pool);
-    pool->log_callback_func(MEMPOOL_LOG_INFO, log_mem_stats);
+    pool->log_callback_func(MEMPOOL_LOG_DEBUG, log_mem_stats);
```

Comment:
Better to pass the callback function from s3server.cc in case of DEBUG mode itself, else it will do lot of sprintf, which may impact perf